### PR TITLE
ActivationValue for RabbitMQ scaler

### PR DESCRIPTION
**Pull request by adborroto**
Signed-off-by: Alejandro Dominguez <adborroto90@gmail.com>

Adding activation threshold (minMetricValue) for Rabbitmq. 

Feature PR: [#2831](https://github.com/kedacore/keda/pull/2831)

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.7/scalers/rabbitmq-queue.md
Comment (JorTurFer) [inline content/docs/2.7/scalers/rabbitmq-queue.md:36]: "Activate the scaler" could be confusing, I guess, WDYT about `Message backlog or Publish/sec. rate to scale from/to zero`?

**Review by zroubalik — COMMENTED** [review]
I think we should have a generic section on `activation threshold` in the docs. Where we describe the behavior (current and the new). And here reference this doc.
WDYT? @kedacore/keda-contributors

**Comment by adborroto** [comment]
> I think we should have a generic section on `activation threshold` in the docs. Where we describe the behavior (current and the new). And here reference this doc. WDYT? @kedacore/keda-contributors

I like it, since, is going to be a generic concept for all scalers.

**Comment by JorTurFer** [comment]
+1 to a specific section

**Comment by adborroto** [comment]
> I think we should have a generic section on `activation threshold` in the docs. Where we describe the behavior (current and the new). And here reference this doc. WDYT? @kedacore/keda-contributors

What do you think about add the section in a different PR/ticket. Since this is only affecting a single scaler. So we don't block this PR and also because not sure where this new section should be placed.

**Comment by zroubalik** [comment]
A new PR is fine, but this general section should be added before this feature lands.

@kedacore/keda-contributors ideas on the docs section?

**Comment by JorTurFer** [comment]
Maybe we can explain it under [Scaling Deployments, StatefulSets & Custom Resources](https://keda.sh/docs/2.6/concepts/scaling-deployments/) due to this only applies to ScaledObjects.
Something like "Differences between activation and scaling"

File: content/docs/2.7/scalers/rabbitmq-queue.md
Comment (JorTurFer) [inline content/docs/2.7/scalers/rabbitmq-queue.md:36]:
```suggestion
- `activationValue` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional, This value can be a float)
```

Comment (JorTurFer) [reply content/docs/2.7/scalers/rabbitmq-queue.md:36]: For the documentation, we are using the same definition for all the scalers, I have suggested it

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**
